lib/server/services/dataPasses/DataPassService.js

```diff
@@ -95,6 +103,50 @@ class DataPassService {
         });
     }
 
+    /**
+     * Set default list of DPL detectors which flags contribute to GAQ (global aggregated quality) for given runs and data pass
+     * Where, for given run, default list of GAQ detectors is intersection of detectors linked to it with list of detectors
+     *  from @see DEFAULT_GAQ_DETECTORS_FOR_PDP_BEAM_TYPE, which depends on beam type of the run
+     *
+     * @param {number} dataPassId data pass id
+     * @param {number[]} runNumbers list of run numbers
+     * @param {object} [options] additional options
+     * @param {sequelize.Transaction} [options.transaction] sequelize transaction
+     * @return {Promise<void>} promise resolved once association between data pass, runs and detectors was set
+     */
+    async setDefaultGaqDetectors(dataPassId, runNumbers, { transaction } = {}) {
```

Comment:
I think that the naming here is misleading, I would rather name it `useDefaultGaqDetectors`

Reply:
applied